backend/schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS "users"(
+    "userId" VARCHAR(10) NOT NULL,
+    "userName" VARCHAR(40) NOT NULL,
```

Comment:
`name` instead of `userName` and similarly please remove table name from the columns as they are contextual.

Reply:
Sure!